src/components/IterableEditComponent.tsx

```diff
@@ -323,53 +319,35 @@ export class IterableEditComponent extends React.Component<ImageUploadProps | It
-                            return <div key={index}
-                                style={this.props.field.iterabletype.style && this.props.field.iterabletype.style.border === "none" ? {} : {
-                                    border: "1px solid #EEE",
-                                    padding: "10px",
-                                    marginTop: "10px"
-                                }}
-                                onMouseEnter={this.showIterableButtons.bind(this, index)}
-                                onMouseLeave={this.hideIterableButtons.bind(this, index)}>
-                                {this.props.field.iterabletype.nestedIterableCollapse && this.props.field.iterabletype.nestedIterableCollapse.title &&
-                                    <div onClick={this.collapseNestedToggle.bind(this, index)} style={titleStyle}>
-                                        <div style={{ display: "inline-block", width: "90%"}}>
-                                            {this.getIterableNestedTitle(index)}
-                                        </div>
-                                        {!this.state.collapsedIndex[index] &&
-                                            <span style={{ marginLeft: "10px", color: "grey", cursor: "pointer" }}
-                                                className="glyphicon glyphicon-chevron-up" aria-hidden="true"/>}
-                                        {this.state.collapsedIndex[index] &&
-                                            <span style={{ marginLeft: "10px", color: "grey", cursor: "pointer" }}
-                                                className="glyphicon glyphicon-chevron-down" aria-hidden="true"/>}
-                                    </div>}
-                                <div style={this.state.collapsedIndex[index] ? { display: "none" } : { display: "inline-block" }}>
-                                    <NestedEditComponent
-                                        readonly={this.props.field.iterabletype.readonly === true || this.props.readonly}
-                                        currentModel={this.state.model[index]}
-                                        fetch={this.props.fetch}
-                                        field={this.props.field.iterabletype}
-                                        additionalModels={this.props.additionalModels}
-                                        modelChanged={this.fieldChanged(index).bind(this, undefined)}
-                                        showTitle={false}
-                                        indent={false}
-                                        modalType={this.props.modalType}
-                                        parentModel={parentModel}
-                                    />
-                                </div>
-                                {this.iterableButtons(index, totalLength)}
-                            </div>
+                            return <IterableNestedComponent
```

Comment:
Converted to New Component (iterable Nested). Handled mouseEnter and mouseLeave action in its state.